rfcs/0168-feature-parameter-names.md

```diff
@@ -0,0 +1,559 @@
+---
+feature: feature_parameter_names
+start-date: 2024-01-10
+author: Dmitry Bogatov
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Nixpkgs usually exports compile-time options of the upstream build system
```

Comment:
Maybe something more specific than «usually» should be said, as the majority of packages exposes no options (which happens sometimes because of upstream not having feature flags and sometimes because the maintainers do not see the benefits as sufficient to justify the effort).